--- a/testbed/testcms2.h	2020-01-01 18:54:10.535763600 -0500
+++ a/testbed/testcms2_mingw.h"	2020-01-01 19:31:36.210523000 -0500
@@ -30,11 +30,13 @@
 #ifdef _MSC_VER
 #    define _CRT_SECURE_NO_WARNINGS 1
 #     include "crtdbg.h"
-#     include <io.h>
 #endif
 
-#include "lcms2_internal.h"
 
+#include "lcms2_internal.h"
+#ifdef CMS_IS_WINDOWS_
+#     include <io.h>
+#endif
 #define cmsmin(a, b) (((a) < (b)) ? (a) : (b))
 
 // Used to mark special pointers
